README.md

```diff
@@ -1,111 +1,117 @@
-# Ascend NPU plugin for vLLM
+<p align="center">
+  <picture>
+    <img alt="vllm-ascend" src="https://github.com/user-attachments/assets/16e71f51-f127-410d-b821-4fdc9ea8adac" width=60%>
+  </picture>
+</p>
 
-## Use Docker
+<h3 align="center">
+vLLM Ascend Backend Plugin
+</h3>
 
-### 1. Download vllm and vllm_ascend
+<p align="center">
+| <a href="https://www.hiascend.com/en/"><b>About Ascend</b></a> | <a href="https://slack.vllm.ai"><b>Developer Slack (#sig-ascend)</b></a> |
+</p>
 
-```bash
-git clone https://github.com/cosdt/vllm-ascend
-cd vllm-ascend
-git clone https://github.com/cosdt/vllm -b apply_plugin
-```
+---
+*Latest News* 🔥
 
-### 2. Build Docker Image
+- [2024/12] We are working with the vLLM community to support [[RFC]: Hardware pluggable](https://github.com/vllm-project/vllm/issues/11162).
+---
+## Overview
 
-> [!NOTE]
-> Modify the base image in Dockerfile according to your device. 
-> More choices could be found at https://hub.docker.com/r/ascendai/cann
+The vllm-ascend is a Ascend backend plugin for vLLM enabling users to run vLLM on Ascend NPU. 
 
-```bash
-docker build -t vllm-npu .
-```
+This is the recommend way the vLLM community supports the Ascend backend, it follows [[RFC]: Hardware pluggable](https://github.com/vllm-project/vllm/issues/11162), through a decoupled way to integrate the Ascend NPU plug-in into vLLM.
 
-### 3. Run docker container
+This enables the most popular open-source models, including Transformer-like, Mixture-of-Expert, Embedding, Multi-modal LLMs to run seamlessly on Ascend NPU.
 
-> [!NOTE]
-> Modify `--device /dev/davinci0` according to your device.
+## Prerequisites
+### Support Devices
+- Atlas A2 Training series (Atlas 800T A2, Atlas 900 A2 PoD, Atlas 200T A2 Box16, Atlas 300T A2)
+- Atlas 800I A2 Inference series (Atlas 800I A2)
 
-```bash
-docker run -dit -v /usr/local/dcmi:/usr/local/dcmi -v /usr/local/bin/npu-smi:/usr/local/bin/npu-smi -v /usr/local/Ascend/driver:/usr/local/Ascend/driver -v /etc/ascend_install.info:/etc/ascend_install.info --device /dev/davinci0 --device /dev/davinci_manager --device /dev/devmm_svm --device /dev/hisi_hdc --shm-size 16G --name vllm vllm-npu:latest bash
-```
-### 4. Enter the container
+### Dependencies
```

Comment:
Not sure this should be a separate doc in somewhere like `docs/environment.md` and link here? Because
- the Ascend env preparation not a easy thing with simple word.
- readme should keep simple

Reply:
Move to a separate file